src/ci/BitbucketPipelinesAdapter.js

```diff
@@ -25,46 +26,76 @@ const BitbucketPipelinesAdapter: CIAdapter = async (config: Config) => {
   };
 
   return {
-    processStatusWebhook(body: any): StatusEvent | null {
+    processStatusWebhook(body: JSONValue): StatusEvent | null {
+      if (
```

Comment:
Should probably separate out some of these checks to give better error messages?